src/configs/dbConfig.ts

```diff
@@ -1,25 +1,14 @@
-import { DataSource } from 'typeorm';
-import { DB_PASSWORD, DB_HOST, DB_PORT, DB_USER, DB_NAME } from './envConfig';
+import { DataSource } from 'typeorm'
+import { DB_PASSWORD, DB_HOST, DB_PORT, DB_USER, DB_NAME } from './envConfig'
+
 export const dataSource = new DataSource({
   type: 'postgres',
   host: DB_HOST,
-  port: Number(DB_PORT) || 5432,
+  port: Number(DB_PORT) ?? 5432,
```

Comment:
For this, I think `||` is the option we should use because `??` assigns the default value only if DB_PORT is null or undefined. `||` assigns the default value if DB_PORT is any falsy value (null, undefined, 0, empty string or false).

(Btw, can't we do it within the envConfig file itself? So we can keep a clean dbConfig file.)

Reply:
This is actually a typescript-enforced lint rule. https://typescript-eslint.io/rules/prefer-nullish-coalescing/

Reply:
Oh right. Then it's okay.